sections/staking/hooks/useClearDebtCalculations.ts

```diff
@@ -1,36 +1,31 @@
 import synthetix from 'lib/synthetix';
-import { NumericValue, toBigNumber, zeroBN } from 'utils/formatters/number';
 import use1InchQuoteQuery from 'queries/1inch/use1InchQuoteQuery';
 import use1InchSwapQuery, { SwapTxData } from 'queries/1inch/use1InchSwapQuery';
 import { ethAddress } from 'constants/1inch';
-import { BigNumber } from 'bignumber.js';
+import Wei, { wei } from '@synthetixio/wei';
 
 type ClearDebtCalculations = {
 	needToBuy: boolean;
-	debtBalanceWithBuffer: NumericValue;
-	missingSUSDWithBuffer: NumericValue;
-	quoteAmount: NumericValue;
+	debtBalanceWithBuffer: Wei;
+	missingSUSDWithBuffer: Wei;
+	quoteAmount: Wei;
 	swapData: SwapTxData | null;
 };
 
 const useClearDebtCalculations = (
-	debtBalance: BigNumber,
-	sUSDBalance: BigNumber,
+	debtBalance: Wei,
+	sUSDBalance: Wei,
 	walletAddress: string
 ): ClearDebtCalculations => {
-	const needToBuy = debtBalance.minus(sUSDBalance).isPositive();
-	const debtBalanceWithBuffer: NumericValue = toBigNumber(
-		debtBalance.plus(debtBalance.multipliedBy(0.0005)).toFixed(18)
-	);
-	const missingSUSDWithBuffer: NumericValue = toBigNumber(
-		debtBalanceWithBuffer.minus(sUSDBalance).toFixed(18)
-	);
+	const needToBuy = debtBalance.sub(sUSDBalance).gt(0);
+	const debtBalanceWithBuffer = debtBalance.add(debtBalance.mul(0.0005));
```

Comment:
we should make ```0.0005``` a constant

Reply:
we could simplify this calculation as well by just doing `debtBalance.mul(1.0005)`, no reason to add